spec/factories/team.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
     factory :team do
-        name {Faker::Esport.team}
+        name    
```

Comment:
This factory will not work, since you did not specify a value for `name`. When you attempt to create an object, the following will appear:
`KeyError (Trait not registered: "name" referenced within "team" definition)`

Not specifying a value for `league` works because `league` already has a factory associated with it.

You could specify a value for `name` or remove it entirely.